src/runtime/tests/schema-test.ts

```diff
@@ -443,4 +444,86 @@ describe('schema', () => {
     assert.strictEqual(schema.fields.age.type, 'Number');
     assert.strictEqual(schema.fields.custom.type, 'Bytes');
   });
+
+
+  const partialOrderGreaterThan = (testCase) => {
+    const [caseName, first, second] = testCase;
+    it(`partially orders entities: ${caseName}`, () => {
+      assert.isTrue(first.isMoreSpecificThan(second));
+      assert.isFalse(second.isMoreSpecificThan(first));
+    });
+  };
+
+  const notPartialOrdered = (testCase) => {
+    const [caseName, first, second] = testCase;
+    it(`determines entities have no overlap: ${caseName}`, () => {
+      assert.isFalse(first.isMoreSpecificThan(second));
+      assert.isFalse(second.isMoreSpecificThan(first));
+    });
+  };
+
+  const partiallyOrderedTestCases = async () => {
```

Comment:
These tests have little to do with the bug you are fixing, please remove them from this patch. If you are testing `isMoreSpecificThan` please do it in a separate PR.

Reply:
Ok. I'll move them to another PR. I'll clean them up first.